scripts/l2ps-stress-test.ts

```
@@ -0,0 +1,375 @@
+
+    // Run stress test
+    console.log(`\n🔥 Starting stress test...`)
+    const startTime = Date.now()
+    const results: TxResult[] = []
+    let successCount = 0
+    let failCount = 0
+
+    for (let i = 0; i < options.count; i++) {
+        // Pick random sender and receiver (different wallets)
+        const senderIdx = i % walletConnections.length
+        let receiverIdx = (senderIdx + 1 + Math.floor(Math.random() * (walletConnections.length - 1))) % walletConnections.length
```

ESLint

[error] 'receiverIdx' is never reassigned. Use 'const' instead.

(prefer-const)